MCSync/FancyUX.cs

```diff
@@ -0,0 +1,161 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+using System.Windows.Forms;
+using System.Drawing;
+using System.Drawing.Drawing2D;
+using System.ComponentModel;
+
+namespace MCSync
+{
+    public class FancyButton : Button
+    {
+        //Fields
+        private int borderSize = 0;
+        private int borderRadius = 0;
+        private Color borderColor = Color.PaleVioletRed;
+
+        //Properties
+        [Category("RZ's Fancy Buttons")]
+        public int BorderSize
+        {
+            get { return borderSize; }
+            set
+            {
+                borderSize = value;
+                this.Invalidate();
+            }
+        }
+
+        [Category("RZ's Fancy Buttons")]
+        public int BorderRadius
+        {
+            get { return borderRadius; }
+            set
+            {
+                borderRadius = value;
+                this.Invalidate();
+            }
+        }
+
+        [Category("RZ's Fancy Buttons")]
+        public Color BorderColor
+        {
+            get { return borderColor; }
+            set
+            {
+                borderColor = value;
+                this.Invalidate();
+            }
+        }
+
+        [Category("RZ's Fancy Buttons")]
+        public Color BackgroundColor
+        {
+            get { return this.BackColor; }
+            set { this.BackColor = value; }
+        }
+
+        [Category("RZ's Fancy Buttons")]
+        public Color TextColor
+        {
+            get { return this.ForeColor; }
+            set { this.ForeColor = value; }
+        }
+
+        public int BorderSize1 { get => borderSize; set => borderSize = value; }
+        public int BorderRadius1 { get => borderRadius; set => borderRadius = value; }
+        public Color BorderColor1 { get => borderColor; set => borderColor = value; }
```

Comment:
These can probably be removed - I don't see any references to them anywhere

Reply:
Those variables are auto-generated and the class breaks without them.